packages/color-scale-block/src/types.ts

```diff
@@ -0,0 +1,121 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MouseEvent } from 'react';
+import { Color, ColorFormat, DraggableItem, DropZonePosition, Palette } from '@frontify/fondue';
+
+export type Settings = {
+    customHeight: boolean;
+    heightSlider: string;
+    heightInput: string;
+    ['color-input']: ColorProps[];
+};
+
+export type FormattedColor = {
+    id?: number | null;
+    alpha?: number | undefined | null;
```

Comment:
`undefined` is not needed.

Reply:
The `FormattedColor` type has been removed, so not actual anymore.